src/lib/components/explorer/results/ResultsPanel.svelte

```diff
@@ -104,8 +104,11 @@
     hasFilterOrExport;
 
   onMount(async () => {
-    unsubFilters = filters.subscribe(() => {
+    unsubFilters = filters.subscribe((filterArray) => {
```

Comment:
This feels like the wrong place to do this, but it does not work in the `Filter.ts` file, because that gets loaded at various times throughout the login and when it initializes with empty filters that would get written. Thoughts/ideas?